FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.DataClient/Models/DicomApiOption/ChangeFeedLatestOptions.cs

```diff
@@ -0,0 +1,24 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System.Collections.Generic;
+using Microsoft.Health.Fhir.Synapse.DataClient.Api.Dicom;
+
+namespace Microsoft.Health.Fhir.Synapse.DataClient.Models.DicomApiOption
+{
+    public class ChangeFeedLatestOptions : BaseApiOptions
```

Comment:
Add `Dicom` prefix to the `ChangeFeedLatestOptions`, and same for other options

Reply:
Same as above.